source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3/package.json

```diff
@@ -61,6 +61,7 @@
     "@aws-cdk/aws-logs": "0.0.0",
     "@aws-solutions-constructs/core": "0.0.0",
     "@aws-solutions-constructs/aws-kinesisfirehose-s3": "0.0.0",
```

Comment:
Can we drop this line?

Reply:
Nope since we create EventbridgeToKinesisFirehoseToS3 in the wrapper